contrib/go-pg/pg/pg_go.go

```diff
@@ -0,0 +1,55 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package pg
+
+import (
+	"context"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	"github.com/go-pg/pg/v10"
+)
+
+// Hook wraps the given pg.Connect with QueryHooks.
+// go-pg support QueryHook, and this is used for achieve tracing sql.
+func Hook(db *pg.DB) *pg.DB {
```

Comment:
Let's rename this to `Wrap`. It is better. Otherwise one may think a hook is returned.

```suggestion
// Wrap augments the given DB with tracing.
func Wrap(db *pg.DB) {
```
We can remove the return value, it serves no purpose.